src/components/forms/OnboardingAbout.tsx

```diff
@@ -49,6 +55,7 @@ export default function OnboardingAbout() {
             {errors.qTwo && <span>{errors.qTwo.message}</span>}
           </div>
         </div>
+        <button type="submit">TESTE ENVIAR</button>
       </Form>
       <pre>
         <code>{JSON.stringify(values, null, 2)}</code>
```

Comment:
I believe this code could be passed to a const
`JSON.stringify(values, null, 2)`

Reply:
Thanks for the comment Nicolas! This is only a temporary, to show the data. Should be removed with the next pull.

Reply:
Thanks for the comment Raysson! This is only a temporary, to show the data. Should be removed with the next pull.